lib/cherry_silt/mongo.rb

```diff
@@ -0,0 +1,33 @@
+# Cherry Silt -- MUD written in ruby using eventmachine
+# Copyright (C) 2015 Sean Hardy
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+require 'mongo'
+
+##
+module CherrySilt
+  ##
+  module Util
+    ##
+    class DB
+      @@client = nil
+
+      def self.find(query, collection)
+        @@client ||= Mongo::Client.new(['127.0.0.1:27017'], database: 'mud')
```

Comment:
I'm thinking pulling line 28 out into it's own static method and just calling it before eventmachine startup. not sure which is a better way to do it. 

This way we'll have to have this line in every query method we have. which I don't like